eventmesh-common/src/main/java/org/apache/eventmesh/common/file/WatchFileTask.java

```diff
@@ -66,11 +66,6 @@ public WatchFileTask(String directoryPath) {
             path.register(this.watchService, StandardWatchEventKinds.OVERFLOW, StandardWatchEventKinds.ENTRY_MODIFY,
                 StandardWatchEventKinds.ENTRY_CREATE, StandardWatchEventKinds.ENTRY_DELETE);
         } catch (IOException ex) {
-            try {
-                this.watchService.close();
-            } catch (IOException e) {
-                ex.addSuppressed(e);
-            }
```

Comment:
However, you removed the part where the `watchService` is closed in the exception handling, and I think that this part should be retained.